tools/hangar/src/utils.ts

```diff
@@ -25,18 +25,13 @@ export async function runWingCommand(options: RunWingCommandOptions) {
       env: options.env,
     }
   );
-  if (options.expectStdErr) {
+
+  if (options.expectFailure) {
     expect(out.exitCode).not.toBe(0);
-    expect(out.stderr).not.toBe("");
   } else {
-    if (
```

Comment:
This change looks correct, but IIRC this was added by @yoav-steinberg to test panic in a consistent way. Is that still working after you remove this `if` statement and reverse the `expectFailure` flags in `error.test.ts` and `invalid.test.ts`?

Reply:
I believe it works.